src/main/java/com/ericsson/ei/controller/model/QueryResponseEntity.java

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Ericsson Copyright header need to be added in top of the file.

Reply:
This is not possible since these are generated based on RAML Schemas. No file in the model package has the header.

Reply:
ok